app/views/templates/news_article.html.haml

```diff
@@ -3,26 +3,30 @@
 
 - breadcrumb :public_news_article, node
 
-%div.list-horizontal
-  = render partial: 'templates/related_sections', locals: {node: node}
-
 %h1
   =node.name
 
 %div.abstract
   %p
     =node.short_summary
     - if node.summary
-      != markdown_content node.summary
+      .summary
+        != markdown_content node.summary
 
 %ul.announcement-info
   - if node.release_date
-    %li Published date: #{l(node.release_date, format: :news)}
+    %li
+      %span
+        Date published
+      #{l(node.release_date, format: :news)}
   %li
-    Published by:
-    %a= link_to(node.section.name, public_node_path(node.section.home_node))
-
+    %span
+      Published by
+    %a= link_to(node.section.name, section_news_articles_path(section: @section.home_node.slug))
 
 = render partial: 'templates/content_body', object: node.content_body
 
 =@hierarchy
+
+-#%div.list-horizontal
+  = render partial: 'templates/related_sections', locals: {node: node}
```

Comment:
I remember needing the `list-horizontal` class to get the tags to render correctly. Plus, this relegates the related sections to the very end of the document when I would have thought it should be near the "published by" entry. Has this been tested and a decision made to put them here?